src/main/java/com/dougnoel/sentinel/files/CsvFile.java

```diff
@@ -0,0 +1,334 @@
+package com.dougnoel.sentinel.files;
+
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+
+import com.dougnoel.sentinel.exceptions.FileException;
+import com.dougnoel.sentinel.strings.SentinelStringUtils;
+import org.apache.commons.csv.CSVFormat;
+import org.apache.commons.csv.CSVParser;
+import org.apache.commons.csv.CSVPrinter;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class CsvFile extends TestFile{
+    private static final Logger log = LogManager.getLogger(CsvFile.class.getName()); // Create a logger.
+
+    private final CSVFormat csvFormat;
+    private final int numHeaderRows;
+
+    /**
+     * Create default CSV file from most recently-downloaded path.
+     * @throws FileNotFoundException in the case that a file does not exist at that path.
+     */
+    public CsvFile() throws FileNotFoundException {
+        this(1);
+    }
```

Comment:
This constructor isn't used. I'd like to see a test for this even if it's just injecting the test file into the correct value in the Download Manager.